ES/ESmy016.xml

```diff
@@ -31,325 +31,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-016</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Dǝrsanä sanbata krǝstiyān “Homily on the Sabbath of Christians”
-                     (Homily on the observance of Sunday, ascribed to Jacob of Serug) (fols. 1r-33v,
-                     38r-40v), incomplete and misplaced&#xD; II) Malkǝʾa sanbata krǝstiyān “Image of
-                     the Sabbath of Christians”: 25 strophes (fols. 33v-37v, 41r-41v,
-                     43r-44r).</summary>
-                  <msItem xml:id="ms_i1">
-                     <locus from="fols. 1r" to="33v, 38r-40v">fols. 1r-33v, 38r-40v</locus>
-                     <title type="incomplete">I. Dǝrsāna sanbata krǝstiyān “Homily on the Sabbath of
-                        Christians” (Homily on the observance of Sunday, ascribed to Jaqob of
-                        Serug)</title>
+                <summary/>
+                         <msItem xml:id="ms_i1">
+                          <locus from="1r" to="33v"/>
+                            <locus from="38r" to="40v"/>
+                          <title type="incomplete" ref="LIT1298Dersan"/>
+                          <incipit xml:lang="gez">
+                             <locus target="#1r"/><hi rend="rubric">በስመ፡ አብ፡ ወወ<gap reason="lost"/>፡ ወመንፈስ፡ ቀዱስ፡
+                             <gap reason="lost"/>ምላክ፡ ድ</hi>ርሳን፡ ዘደረሰ፡ አባ፡ ያዕቆብ፡ ዘሥሩግ፡ በእንተ፤
+                             ሰንበተ፤ ክርስቲያን፤ ጸሎታ፡ ወበረከ<hi rend="rubric">ታ፡ የሃሉ፡ ምስለ፡ ገብራ፡</hi>
+                             <space reason="rubrication" unit="chars" quantity="3"/><hi rend="rubric">ለዓለመ፡ ዓለም</hi>፨ ስምዑ፡ ፍቁራንየ፡ ውሉደ፡
+                             ቤተ፡ ክርስቲያን፤ ዕበያ፡ ወክብራ፡ ለሰንበተ፡ ክርስቲያን፤ አብ፡ ቀደሳ፡ ወልድ፡ ባረካ፡ መንፈስ፡ ቅዱስ፡ አልዓላ፡ <foreign xml:lang="en">
+                            <locus target="#1r"/>In the name of the Father and the Son and
+                            the Holy Spirit, one God. The homily that ʾabbā Jaqob of Serug
+                            composed on the Sabbath of Christians. May her prayer and her blessing be
+                              with her servant <space reason="rubrication" unit="chars" quantity="3"/> forever
+                              and ever. Listen, my beloved ones, sons of the
+                            church, anout the greatness and honour of the Sabbath of Christians that the Father sanctified,
+                          and the Son blessed, and the Holy Spirit elevated... </foreign>
+                          </incipit>
+                          <explicit xml:lang="gez">
+                             <locus target="#33r"/>ኦወልደ፡ እግዚአብሔር፡ ሕያው፡ ዕቀቦ፡ ለገብርከ፡<space reason="rubrication" unit="chars" quantity="3"/> ወ<locus target="#33v"/>
```

Comment:
locus element must be only one at the beginning
`<locus from="33r" to "33v">`, for the page break insert `<pb/>`

Reply:
ok